### ISSUE TITLE
Bi-coloured two-way lollipop accessibility

### ISSUE THREAD

**Issue by xenatisch** [publichealthengland/coronavirus-dashboard#318]
Bi-coloured lollipop chart markers are not accessible to users with colourblindness (red / green). 

Based on recommendations obtained from users, it is recommended that different markers are used to complement the colours.
